src/sentry/tsdb/snuba.py

```diff
@@ -760,7 +760,7 @@ def get_range(
         return {k: sorted(result[k].items()) for k in result}
 
     def get_distinct_counts_series(
```

Comment:
Could we update the base class method signatures as well: src/sentry/tsdb/base.py

Reply:
good call